biom/cli/table_subsetter.py

```diff
@@ -64,7 +64,10 @@ def subset_table(input_hdf5_fp, input_json_fp, axis, ids, output_fp):
             input_json_fp = f.read()
 
     with open(ids, 'U') as f:
-        ids = [line.strip() for line in f]
+        ids = []
```

Comment:
Minor but isn't this the same?
```python
ids = [line.strip().split('\t')[0] for line in f if not line.startwith('#')
```
you know to avoid appends (no need to change if you think it makes the code less readable.